### PR TITLE
Sheets node name

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Polished first pass on this doc rewrite.

### CODE REVIEW

File: api/sheets/get_name.md [hunk 1]
```diff
@@ -6,40 +6,36 @@ parent: Google Sheets
...
 
-## `=DCGETNAME(dcids)`
+Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
 
-Given a cell or a column range of cells with DCIDs, return the names of those dcids.
+## General information about this method
 
-**Arguments**
-*    `dcids` - DCIDs to get the names of
+**Signature**: `=DCGETNAME(dcids)`
```
Comment (beets) [inline api/sheets/get_name.md:15]: Should these be "Formula" instead for Sheets?

File: api/sheets/get_name.md [hunk 2]
```diff
@@ -6,40 +6,36 @@ parent: Google Sheets
...
 
 ## Examples
 
-In this example, we find the names of the Data Commons nodes for [California](https://datacommons.org/browser/geoId/06) and [Alameda County](https://datacommons.org/browser/geoId/06001). The function call in the image uses a single call to get the names of "geoId/06" and "geoId/06001", but it is equivalent to typing:
+Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
 
-```
-=DCGETNAME("geoId/06")
-```
+### Example 1: Retrieve the common name of the country of Côte d'Ivoire.
```
Comment (beets) [inline api/sheets/get_name.md:25]: Maybe leave "Cote d'Ivoire" out from the header here, so as not to confuse between this and "common" name
```suggestion
### Example 1: Retrieve the name of a country by it's DCID.
```

**Review by beets — CHANGES_REQUESTED** [review]

File: api/sheets/get_name.md (hunk 1 above)
Comment (KilimAnnejaro) [reply api/sheets/get_name.md:15]: Yep, that is a GREAT idea!

File: api/sheets/get_name.md (hunk 2 above)
Comment (KilimAnnejaro) [reply api/sheets/get_name.md:25]: This is a great idea! I've incorporated it.

**Review by beets — APPROVED** [review]

**Pull request merged**